core/src/main/java/com/oath/oak/OakWBuffer.java

```diff
@@ -15,12 +15,6 @@
  */
 public interface OakWBuffer {
```

Comment:
Can we improve the lack of symmetry in naming: OakRKeyBufferImpl vs OakWBufferImpl. "W" should be for values, right? Or we'll have some other "R" for read-only values?

Reply:
R: read only
W: wriyable
The read buffers have different implementation for key/value, but the write buffer have only one implementation for both.